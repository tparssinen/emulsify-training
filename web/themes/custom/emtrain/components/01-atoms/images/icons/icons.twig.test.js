import { join } from 'path';
import { render, Twig } from 'twig-testing-library';

import loadYaml from '../../../../util/loadYaml';
import { setupTwig, namespaces } from '../../../../.storybook/setupTwig';

setupTwig(Twig);

describe('icons', () => {
  it('can render an icon', async () => {
    const { container } = await render(
      join(__dirname, 'icons.twig'),
      loadYaml(join(__dirname, 'icons.yml')),
      namespaces,
    );

    expect(container).toMatchSnapshot();
  });
});
