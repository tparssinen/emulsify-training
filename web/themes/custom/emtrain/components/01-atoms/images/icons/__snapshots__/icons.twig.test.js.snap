// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icons can render an icon 1`] = `
<div>
  <div
    class="icons-demo"
    id="icons"
  >
    
      
    <div
      class="icons-demo__item"
    >
      
      






      <svg
        class="icon"
      >
        
      
        <use
          xlink:href="icons.svg#facebook"
        />
        

      </svg>
      
      
      <pre>
        facebook
      </pre>
      
    
    </div>
    
      
    <div
      class="icons-demo__item"
    >
      
      






      <svg
        class="icon"
      >
        
      
        <use
          xlink:href="icons.svg#instagram"
        />
        

      </svg>
      
      
      <pre>
        instagram
      </pre>
      
    
    </div>
    
      
    <div
      class="icons-demo__item"
    >
      
      






      <svg
        class="icon"
      >
        
      
        <use
          xlink:href="icons.svg#menu"
        />
        

      </svg>
      
      
      <pre>
        menu
      </pre>
      
    
    </div>
    
      
    <div
      class="icons-demo__item"
    >
      
      






      <svg
        class="icon"
      >
        
      
        <use
          xlink:href="icons.svg#twitter"
        />
        

      </svg>
      
      
      <pre>
        twitter
      </pre>
      
    
    </div>
    
  
  </div>
  

  <br />
  

</div>
`;
